packages/layout/src/tree-layout/layout-component/dummy-component.js

```diff
@@ -0,0 +1,44 @@
+import LayoutComponent from './layoutComponent';
+
+class DummyComponent extends LayoutComponent {
+    getLogicalSpace () {
+        return {
+            width: this.boundBox().width,
+            height: this.boundBox().height
+        };
+    }
+
+    setSpatialConfig (conf) {
+        this.boundBox({ top: conf.y, left: conf.x });
+        this.newDimensions = {
+            width: conf.width,
+            height: conf.height
+        };
+        this.renderAt(conf.renderAt);
```

Comment:
Class method must always return something.
If nothing is to be returned, return "this"